src/constants/rteBlocks.ts

```diff
@@ -0,0 +1,5 @@
+export const RTE_BLOCKS = {
+  ACCORDION: "accordion",
+  CARDGRID: "cardgrid",
+  DIVIDER: "divider",
+}
```

Comment:
nit: can we add the `as const` annotation here to prevent any edits to this data structure

Reply:
@seaerchin sure, added in b28f0e0daf9e2b01cde1c9295b161fb9bc2e3908